docs/sphinx/user_guide/using_raja.rst

```diff
@@ -32,23 +32,10 @@ Then, pass the path of RAJA to CMake when you configure your code::
 
   cmake -DRAJA_DIR=<path-to-raja>/share/raja/cmake
 
-The ``RAJA-config.cmake`` file provides the following variables:
-
-======================   ===================================
-Variable                 Default
-======================   ===================================
-``RAJA_INCLUDE_DIR``     Include directory for RAJA headers.
-``RAJA_LIB_DIR``         Library directory for RAJA.
-``RAJA_COMPILE_FLAGS``   C++ flags used to compile RAJA.
-``RAJA_NVCC_FLAGS``      CUDA flags used to compile RAJA.
-======================   ===================================
-
-It also provides the ``RAJA`` target, that can be used natively by CMake to add
-a dependency on RAJA. For example::
+The ``RAJA-config.cmake`` file provides a ``RAJA`` target, that can be used
+natively by CMake to add a dependency on RAJA. For example::
```

Comment:
I think you deleted too much here. We need some or all of that sentence after the table to explain the two CMakeList.txt file lines.

Reply:
I don't think so. The stuff I deleted no longer exists.

How do you think it should read?

Reply:
Nevermind. I see what you did now. I was confused before.